src/resolvers/queries/SystemQuery.ts

```diff
@@ -71,7 +71,7 @@ export class SystemQuery {
       cachedVersion = content.toString().trim();
 
       return cachedVersion;
-    } catch (err) {
+    } catch (err: any) {
```

Comment:
See this comment,
https://github.com/UserOfficeProject/user-office-backend/pull/421#discussion_r725931820
Maybe the same applies here